src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,27 @@
+package core.basesyntax;
+
+import java.util.ArrayList;
+
+public class Main {
+    public static void main(String[] args) {
+        ArrayList<Figure> figures = new ArrayList<>();
```

Comment:
```suggestion
        List<Figure> figures = new ArrayList<>();
```
Use abstractions as a type instead of implementations